src/ResourceManager/DataFactoryV2/Commands.DataFactoryV2/IntegrationRuntimes/SetAzureDataFactoryIntegrationRuntimeCommand.cs

```diff
@@ -164,8 +181,18 @@ public override void ExecuteCmdlet()
                             base.Name),
                         "Type");
                 }
+
+                if (AuthKey != null)
```

Comment:
Confused here.  When should the user pass AuthKey? Looks like always fails.

Reply:
@zhangyd2015

Reply:
User can create linked integration runtime either with AuthKey, or with SharedIntegrationRuntimeResourceId, but can't with both.